README.md

```diff
@@ -1,70 +1,16 @@
-* [individuals-other-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-other-income-api)
-* [individuals-pensions-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-pensions-income-api)
-* [individuals-reliefs-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-reliefs-api)
-* [individuals-savings-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-savings-income-api)
-* [individuals-state-benefits-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-state-benefits-api)
-* [mtd-sa-test-support-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/mtd-sa-test-support-api)
-* [obligations-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/obligations-api)
-* [other-deductions-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/other-deductions-api)
-* [property-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api)
-* [self-assessment-accounts-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-accounts-api)
-* [self-assessment-biss-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-biss-api)
-* [self-assessment-bsas-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-bsas-api)
-* [self-employment-business-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api)
-* [self-assessment-assist (HMRC Assist API)](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-assist)
-
-## Get notified of changes
-
-To be notified when an MTD API change is released to the Sandbox or Production environments, follow these steps while logged-in to your GitHub account:
-
-1. Go to: https://github.com/hmrc/income-tax-mtd-changelog
-2. Click the "Watch" drop-down at the top right, and choose "Custom.. Releases"
-3. Go to: https://github.com/settings/notifications
-4. Under "Subscriptions.. Participating, @mentions and custom", ensure that you have "Email" ticked
-
-You should now receive an email approximately every two weeks containing the changes made to the APIs since the last update.
-
-## Roadmap
-
-For planned changes, see the [Income Tax (Making Tax Digital) roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
+This page contains a log of the latest changes for [Income Tax (Making Tax Digital) APIs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api?filter=income-tax-mtd). For information about planned changes to Income Tax (Making Tax Digital) APIs, see [Income Tax (Making Tax Digital) roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
```

Comment:
Could avoid repeating "Income Tax (Making Tax Digital) APIs" so many times by changing to "For information about planned changes to these APIs, see [our roadmap]."